integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java

```diff
@@ -1077,6 +1077,20 @@ public String testModel2() {
         return "OK";
     }
 
+    @GET
+    @Path("projection")
+    @Transactional
+    public String testProjection() {
+        Assertions.assertEquals(1, Person.count());
+
+        PersonName person = Person.findAll().project(PersonName.class).firstResult();
+        Assertions.assertEquals("2", person.name);
+
+        Assertions.assertEquals(1, Person.findAll().project(PersonName.class).count());
```

Comment:
Can we add some tests with queries and parameters too please?

Reply:
I agree it misses some tests :)